audio/softsynth/fluidsynth.cpp

```diff
@@ -179,7 +184,16 @@ int MidiDriver_FluidSynth::open() {
 
 	const char *soundfont = ConfMan.get("soundfont").c_str();
 
+#ifdef IPHONE_OFFICIAL
+	char *soundfont_fullpath[PATH_MAX];
+	const char *document_path = iOS7_getDocumentsDir();
+	strcpy((char *) soundfont_fullpath, document_path);
+	strcat((char *) soundfont_fullpath, soundfont);
```

Comment:
I suggest to use a `Common::String` as buffer here. Just for future reference: `strcpy`/`strcat` are pretty unsafe, we usually use `Common::strlcpy` and `Common::strlcat` instead, which are implementing OpenBSD's `strlcpy` and `strlcat`.